net/sing-box/Makefile

```diff
@@ -0,0 +1,131 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sing-box
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/SagerNet/sing-box/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=a219061d0ebaa6f2ba0111b28bec4a0665bab029037cac0d0ee51b2b168de0b8
+
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Van Waholtz <brvphoenix@gmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/sagernet/sing-box
+GO_PKG_BUILD_PKG:=$(GO_PKG)/cmd/sing-box
+GO_PKG_LDFLAGS_X:=$(GO_PKG).Version=$(PKG_VERSION)
```

Comment:
`Version` is defined under `constant` package.
https://github.com/SagerNet/sing-box/blob/dev/constant/version.go